docs/EN_US/ContainerizedHPCC/ContainerizedMods/CustomConfig.xml

```diff
@@ -992,4 +992,154 @@ eclagent:
+        key/value pair already exists as a default in the HPCC Systems
+        delivered values.yaml file. The key is referenced in the component's
+        ldap.yaml file. You may override these and add additional key/values
+        as needed.</para>
+
+        <programlisting>  secrets:
+     authn:
+       admincredsmountname: "admincredssecretname"       #exernalize LDAP Admin creds
+       admincredsaltmountname: "admincredsaltsecretname" #exernalize alternate LDAP Admin creds
+</programlisting>
+      </sect3>
+
+      <sect3 id="ENABLE_k8S_LDAP">
+        <title>Enable LDAP Authentication</title>
+
+        <para>In the delivered
+        HPCC-Platform/esp/applications/common/ldap/ldap.yaml file, the
+        "ldapAdminSecretKey" is already set to the key mount name illustrated
+        in the example above. To enable the LDAP authentication and to
+        override this value, override the ESP/ECLWatch helm component located
+        in values.yaml :</para>
+
+        <programlisting>esp:
+- name: eclwatch
+  application: eclwatch
+  auth: ldap
+  ldap:
+    ldapAddress: "myldapserver"
+    ldapAdminSecretKey: "admincredsaltmountname"   # use alternate secrets creds
+</programlisting>
+
+        <para> </para>
+      </sect3>
+    </sect2>
+
+    <sect2 id="Configuring_For_HashoCorpVault">
+      <title>Configuring LDAP for HashiCorp Vault</title>
+
+      <para>To configure secrets for HashiCorp Vault, from it's command line
+      interface, create the LDAP "secret" similar to the following example.
+      Note the secret name, is "myvaultadmincreds" in this example. The LDAP
+      Administrator "username" and "password" key/values are required;
+      additional properties are allowed but ignored. The secret name must be
+      specified with the "secret/authn/" prefix.</para>
+
+      <programlisting>   vault kv put secret/authn/myvaultadmincreds username=hpcc_admin password=t0pS3cr3tP@ssw0rd
+   vault kv get secret/authn/myvaultadmincreds
+</programlisting>
+
+      <para>This effectively sets the values for the LDAP Administrator's
+      username and password into the HashiCorp Vault.</para>
+
+      <para>For more information about creating secrets for HashiCorp Vault
+      see the appropriate HashiCorp documentation for your
+      implementation.</para>
+
+      <sect3 id="Deploying_HashiCorpVault">
+        <title id="DeployingTheHashiCorpVault">Deploying the HashiCorp
+        Vault</title>
+
+        <para>You can deploy the HashiCorp Vault secrets when you override the
+        "secrets:" section in HPCC-Platform/helm/hpcc/values.yaml, or in your
+        customized configuration chart.</para>
+
+        <para>The Vault name is defined for this example in a
+        values-secrets.yaml configuration chart. You can find an example of
+        this chart in the HPCC-Platform repository under
+        /helm/examples/secrets/values-secrets.yaml.</para>
+
+        <programlisting>  vaults:
+  authn:
```

Comment:
I see the vault name shown here. Is it appropriate the relate that to how the command line above knows the vault name and where to store the secrets? That might tie it together for the reader.